ingeniamotion/motion_controller.py

```diff
@@ -9,12 +9,18 @@
 from ingeniamotion.configuration import Configuration
 from ingeniamotion.drive_tests import DriveTests
 from ingeniamotion.errors import Errors
-from ingeniamotion.fsoe import FSoEMaster
 from ingeniamotion.information import Information
 from ingeniamotion.io import InputsOutputs
 from ingeniamotion.metaclass import DEFAULT_AXIS, DEFAULT_SERVO
 from ingeniamotion.motion import Motion
 
+try:
+    from ingeniamotion.fsoe import FSoEMaster
+except ImportError:
+    FSOE_MASTER_INSTALLED = False
```

Comment:
I would place this on the import of fsoe-master instead of the ingeniamotion to avoid masking other errors

Reply:
d23549a